Sources/TokamakCore/Views/Navigation/NavigationView.swift

```diff
@@ -17,6 +17,8 @@
 
 final class NavigationContext: ObservableObject {
   @Published var destination = NavigationLinkDestination(EmptyView())
+  @Published var title: String? = nil
```

Comment:
<!--
  1 failure:  Initializing an o...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/Navigation/NavigationView.swift;
  Line: 20;
-->
- :no_entry_sign: Initializing an optional variable with nil is redundant. (`redundant_optional_initialization`)